gladys-assistant/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  web:
+    image: gladysassistant/gladys:v4
+    restart: on-failure
+    stop_grace_period: 1m
+    privileged: true
```

Comment:
Why do you need it to be privileged?

Reply:
@cicoub13 do you happen to know if all of the following are required for Gladys functionality:
`priveleged: true`
`cgroup: host`
`- /var/run/docker.sock:/var/run/docker.sock`

Reply:
In particular, I'm wondering how important binding the Docker socket is:
`/var/run/docker.sock:/var/run/docker.sock`

This gives the Gladys container complete control of the Docker daemon, which we can't allow because Gladys can then issue Docker commands to the host's Docker daemon, where other app containers are running.

Reply:
For the **Docker daemon binding**, it's needed from our side to create, restart other containers when a user adds an integration like Zigbee2Mqtt / MQTT / Node-Red.
Gladys will automatically pull, configure and start a new container
We try to be as smooth for the users but still managed integrations cleanly by using docker each time it's needed.
---
For the `cgroup` and `privileged`, let me confirm it with the community/developers.

Reply:
We use the cgroup in Gladys to get Gladys current containerId in Docker ( https://github.com/GladysAssistant/Gladys/blob/master/server/lib/system/system.getGladysContainerId.js#L32 ) 

We use this to check if the container is in the correct state (network_mode = host for example)

privileged is used for some integration like Bluetooth to be able to scan for Bluetooth devices 

Home Assistant Umbrel integration uses it too: https://github.com/getumbrel/umbrel-apps/blob/master/home-assistant/docker-compose.yml#L8

Reply:
Thanks very much for the explanations @cicoub13 and @Pierre-Gilles. Let me run this by Luke to figure out the best path forward.

Reply:
@cicoub13 @Pierre-Gilles - understood regarding `cgroup` and `priveleged: true`. These are fine to keep to maintain app functionality. Thanks for confirming their necessity.

I have added some thoughts on the docker socket mount here: https://github.com/getumbrel/umbrel-apps/pull/1044#issuecomment-2133467901